src/components/analyticVisualization/EditVisualization.js

```diff
@@ -0,0 +1,96 @@
+import React, {useEffect, useState} from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import {
+    Button,
+    ButtonStrip,
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+} from '@dhis2/ui'
+import {NotEditableElement, VisualizationGroup, VisualizationTitle} from "./index";
+
+const EditVisualization = ({open, settings, handleChange, currentGroup, groups, handleEdit, handleClose, type}) => {
+    const [typeName, useTypeName] = useState('')
+    const [label, useLabel] = useState('')
+
+    const chooseTypeName = type => {
+        switch (type) {
+            case 'home':
+                useTypeName('')
+                useLabel('Home')
+                break
+            case 'program':
+                useTypeName('programName')
+                useLabel('Program')
+                break
+            case 'dataset':
+                useTypeName('datasetName')
+                useLabel('Data set')
+        }
+    }
+
+    useEffect(() => {
+        chooseTypeName(type)
+    }, [type])
+
+    return (
+        <>
+            {open && (
+                <Modal position="middle">
+                    <ModalTitle>{i18n.t('Edit {{label}} visualization', {label})}</ModalTitle>
+                    <ModalContent>
+                        { type !=='home' &&
```

Comment:
Hey, 
I believe it is useful to move the string `home` and all the other `type` possible values to a `const` object. WDYT? 
Thanks!

Reply:
Hi, 
currently, I'm only using the `type` home, but I can create a `const` object with `type` constants and the ones that come from the pattern you mentioned before. Like this:

```
const sections = {
    home: {
        type: 'home',
        typeName: '',
        label: i18n.t('Home'),
    },
    program: {
        type: 'program',
        typeName: 'programName',
        label: i18n.t('Program'),
    },
    dataset: {
        type: 'dataset',
        typeName: 'datasetName',
        label: i18n.t('Data set'),
    }
}
```